package.json

```diff
@@ -29,6 +29,7 @@
     "@sentry/nextjs": "^6.19.3",
     "@sentry/react": "^6.19.2",
     "@sentry/tracing": "^6.19.2",
+    "@stitches/react": "^1.2.8",
```

Comment:
question (?): this new install doesn't seem to be used anywhere, do we need it?

Reply:
Oh phew! I was going to comment that adding a new (largish) dep for an animation might be the wrong approach - but in that case, it's easy to remove!

Reply:
I am using it. In ChevronDownIcon

Reply:
alright, i missed that. in that case, let me rephrase, why add a new library to wrap the icon when you could essentially just wrap it in a div, add some tailwind classes to it to handle the transition and the transform?

Reply:
cause in `multiple` type I can't control the active state myself as I did previously. so the Chevron would not rotate anymore when wrapped in a div. I need to control `[data-state=open] &` on the icon (which is added by radix primitives on `multiple`) and it would be great to do this with tailwind but did not get my head around adding the transition when this `[data-state=open] &` is set.  any suggestion?

Reply:
https://www.radix-ui.com/docs/primitives/components/accordion `multiple` is quite different than `single`

Reply:
this is why I did it as in their example in documentation.

Reply:
haven't really done a deep dive in the docs, but looks like this could potentially be resolved with using a controlled state, as before. the docs suggest that the accordion root takes a `value: string[]` type, i assume that would be for multiples. did you try changing the value state from
```
const [value, setValue] = useState<string>('');
```
to:
```
const [value, setValue] = useState<string[]>([]);
```
?

let me know if that is not working, and happy to approve.

Reply:
'value' isavailable only on 'single' type. that was the issue.

Reply:
`onValueChange` is not on `multiple`, so can't set value. 
on `single` I can have only one expansion panel open. I try to open another one and the other one closes. this is why I used this type of chevron (as per their documentation).

Reply:
updated with setting up the array of values as in your suggestion.